lib/msal-browser/src/app/ClientApplication.ts

```diff
@@ -505,6 +505,20 @@ export abstract class ClientApplication {
         this.eventHandler.removeEventCallback(callbackId);
     }
 
+    /**
+     * Adds event listener that emits an event when a user account is added or removed from localstorage in a different browser tab or window
+     */
+    addAccountStorageListener(): void {
```

Comment:
This name makes it seem like you should be able to add a callback, which you can't.

Reply:
Changed to `enableAccountStorageEvents` and `disableAccountStorageEvents`